lib/features/space-tool/SpaceToolPreview.js

```diff
@@ -118,7 +118,7 @@ export default function SpaceToolPreview(
       var dragGroup = svgCreate('g');
       svgAttr(dragGroup, styles.cls('djs-drag-group', [ 'no-events' ]));
 
-      svgAppend(canvas.getDefaultLayer(), dragGroup);
+      svgAppend(canvas.getActivePlane().layer, dragGroup);
```

Comment:
As we're using this across a number of places in our code base, does it make sense to create an API for it?

```
canvas.getActiveLayer('....')
```